spec/type.dd

```diff
@@ -211,10 +211,10 @@ $(SECTION3 $(LEGACY_LNAME2 Usual Arithmetic Conversions, usual-arithmetic-conver
     promotion. For example:)
 
 ---
-ubyte  u1 = cast(byte)-1;  // error, -1 cannot be represented in a ubyte
-ushort u2 = cast(short)-1; // error, -1 cannot be represented in a ushort
-uint   u3 = cast(int)-1;   // ok, -1 can be represented in a uint
-ulong  u4 = cast(long)-1; // ok, -1 can be represented in a ulong
+ubyte  u1 = cast(byte)-1;  // error, -1 cannot be represented in an ubyte
```

Comment:
Original was correct in these cases (the 'u' is pronounced as a consonant). Everything else LGTM.